### ISSUE TITLE
Tests are not being run on CI (TeamCity)

### ISSUE THREAD

**Issue by dmitrijs-balcers** [mad-eye/meteor-mocha-web#196]
Recently I upgraded to meteor 1.2.1 from 1.1.
After that tests on our CI aren't executed anymore.

When I try to run this command on my local machine: 
```
meteor test-packages --velocity --driver-package respondly:test-reporter@1.0.1 --once --verbose
```
Tests also are not being run unless I open this page `http://localhost:3000` from my browser.

Can anybody suggest what might be the cause of this issue and how should I properly run the tests on CI?

**Comment by gabrielpoca** [comment]
Hey @dmitrijs-balcers did you get any updates on this? I'm facing the same issue right now.

**Comment by dmitrijs-balcers** [comment]
@gabrielpoca Hey, the PR which is pending on this issue is postponed for now.
But we really need this to be fixed, otherwise we can not switch to `Meteor 1.2`

**Comment by gabrielpoca** [comment]
Thank you for answering, but I didn't understand, I'm sorry. Which PR is pending?

**Comment by dmitrijs-balcers** [comment]
Oh you don't see the PR because it is from private repo. But anyhow we have one PR in our project, which is postponed due to this issue.

**Comment by idmontie** [comment]
Running into the same problem here. Is there a workaround for getting tests to run on Travis?

**Comment by dmitrijs-balcers** [comment]
Anyone found any solution or workaround on this?

**Comment by gabrielpoca** [comment]
@dmitrijs-balcers I'm waiting for 1.3 . Didn't upgrade my productions apps to 1.2 .
